src/ui/ui_porting.c

```diff
@@ -127,8 +130,11 @@ int lvgl_init_porting() {
 
     lv_disp_drv_init(&disp_drv);
 #else
-    SDL_InitSubSystem(SDL_INIT_VIDEO);
+    if (SDL_WasInit(SDL_INIT_VIDEO) == 0) {
+        SDL_InitSubSystem(SDL_INIT_VIDEO);
+    }
```

Comment:
maybe add else { log it (not that it should ever happen) }

Reply:
Added. It may happen if the input thread managed to initialise first.